kolibri/plugins/coach/assets/src/views/lessons/LessonResourceSelectionPage/ContentCardList.vue

```diff
@@ -60,13 +60,13 @@
 
     <template>
       <KButton
-        v-if="showButton && !loadingMoreState"
+        v-if="showButton"
         :text="coreString('viewMoreAction')"
         :primary="false"
         @click="$emit('moreresults')"
       />
       <KCircularLoader
-        v-if="(viewMoreButtonState === ViewMoreButtonStates.LOADING) & loadingMoreState"
```

Comment:
This loadingMoreState is a redundant prop, as we can manage this with the loading state in `viewMoreButtonState`